main/main.cpp

```diff
@@ -1881,10 +1880,7 @@ bool Main::start() {
 				script = args[i + 1];
 #ifdef TOOLS_ENABLED
 			} else if (args[i] == "--doctool") {
-				doc_tool = args[i + 1];
-				for (int j = i + 2; j < args.size(); j++) {
-					removal_docs.push_back(args[j]);
-				}
+				doc_tool_path = args[i + 1];
```

Comment:
To avoid conflicts with other flags like you've mentioned on RC, can't we check here if `args[i + 1]` starts with `-` and if so, don't use it for path? Paths can't start with `-`, can they? And even if they can, wrapping it is quotation marks should work, right?

Reply:
Yeah that makes sense. Paths can start with `-` (at least on Linux), but indeed wrapping in quotation marks works, and in any case that'd be a pretty far-fetched use case for something which is supposed to be used with `.` in 99.9% of cases.

Reply:
Done.